packages/gitbook/src/intl/translations/en.ts

```diff
@@ -13,6 +13,7 @@ export const en = {
     search_scope_space: '${1}',
     search_scope_all: 'All content',
     search_ask: 'Ask "${1}"',
+    search_ask_description: 'Let GitBook AI answer your question',
```

Comment:
```suggestion
    search_ask_description: 'Ask AI for an answer to your question',
```

I don't think we should mention GitBook here. As these are visitor-facing sites, they carry the branding of the user and the GitBook touch should be as light as possible (keeping it just to the "powered by GitBook" badge).

I'm not sure of the best wording for it, let's discuss internally.

Reply:
That phrasing is a less easy to read. Before changing it since it means updating all translation files I'll ask @stephen-ashby for his copy input

Reply:
Could just be

> Find the answer with AI

Or

> Use AI to answer your question

or even

> Let AI answer your question

Reply:
How about :

> Ask AI to answer your question

Reply:
Actually I think 

> Find the answer with AI

is best. 

@emmerich ok with you or another suggestion?

Reply:
Ok going with "Find the answer with AI"